### DISCUSSION TITLE
Copyright

### DISCUSSION THREAD

**Discussion by devgianlu** [librespot-org/librespot-java#341] [General]
Hi,
I recently added a copyright header to all source files. I encourage everybody that contributed to this project to make a PR or leave a message down here to let me know where they contributed so that I can add their names in there.

Pinging some of you:
@uvjustin @funtax @lkwg82 @josephcarroll @Iscle @fiechtor @mitschwimmer

**Comment by mitschwimmer** [comment]
Thank you for offering that. But I am totally fine with being mentioned somewhere with less maintenance needed in the long run.

**Reply by funtax** [reply]
Thanks @devgianlu . Same here, I am just happy to help somehow.

**Comment by uvjustin** [comment]
Just had a look, only made a few minor changes and added a few metadata functions in Player.java which has been refactored into DacpMetadataPipe.java.
I'm fine with getting mentioned and also with not getting mentioned :-)
